eth/protocols/snap/sync.go

```diff
@@ -31,6 +31,7 @@ import (
 	"github.com/ethereum/go-ethereum/common/math"
 	"github.com/ethereum/go-ethereum/core/rawdb"
 	"github.com/ethereum/go-ethereum/core/state"
+	accounT "github.com/ethereum/go-ethereum/core/state/account"
```

Comment:
Oh no!!

Reply:
oh yes :trollface:

Reply:
seriously though, there is clearly a name clash, if you have a better naming suggestion, I'm happy to update.

Reply:
Actually, I do have a better sugggestion. Essentially, what you're doing is creating a separate package which contains the state `types`. As of right now, there's only one type. So how about having an `account.go` inside `core/state/types`.?